runtime/vm/createramclass.cpp

```diff
@@ -1622,13 +1635,15 @@ loadSuperClassAndInterfaces(J9VMThread *vmThread, J9ClassLoader *classLoader, J9
  * appropriate Java error on the VM.
  */
 static VMINLINE BOOLEAN
-loadFlattenableFieldValueClasses(J9VMThread *vmThread, J9ClassLoader *classLoader, J9ROMClass *romClass, UDATA classPreloadFlags)
+loadFlattenableFieldValueClasses(J9VMThread *vmThread, J9ClassLoader *classLoader, J9ROMClass *romClass, UDATA classPreloadFlags, J9Class **valueTypesOut)
 {
 	struct J9ROMFieldWalkState fieldWalkState;
```

Comment:
can initialize this like `fieldWalkState = {0];`
  
  
Also don't need the `struct` part